services/kvstoremanager/redis.go

```diff
@@ -14,31 +14,24 @@ import (
 var abortableErrors = []string{}
 
 type redisManagerT struct {
-	config types.ConfigT
-	client *redis.Client
+	clusterMode   bool
+	config        types.ConfigT
+	client        *redis.Client
+	clusterClient *redis.ClusterClient
 }
 
 func init() {
 	abortableErrors = []string{"connection refused", "invalid password"}
 }
 
 func (m *redisManagerT) Connect() {
+	m.clusterMode, _ = m.config["clusterMode"].(bool)
```

Comment:
let's make default clusterMode value to true? Like if it's not present in config for some reason.